### ISSUE TITLE
Performance on UCF101 is lower than the reported

### ISSUE THREAD

**Issue by Jamine-W** [tobyperrett/trx#11]
Thanks for sharing your code.  I use your code to train on UCF101 with the suggested hyper-parameters (i.e., lr=0.001, trans_linear_out_dim=1152， img_size=224, tasks_per_batch =16, num_test_tasks=10000) and the same data loader. However, the 5-way 5-shot accuracy can only attain 94.8%, which is lower than your reported 96.1%. Are there any other tricks you used to improve the performance? Or I use the wrong hyper-parameters?

**Comment by frelwx** [comment]
I have the same problem with you, do u solve it ?

**Comment by bedman367** [comment]
> I have the same problem with you, do u solve it ?

I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 .  On my machine, performance on ssv2(OTAM ) after traing 75000 tasks  is 50.5+/-0.4 ,  perfomance on ucf101(ARN)  after training 10000 tasks is 82.0+/-0.3.   Have you solved this problem?   QAQ

**Comment by frelwx** [comment]
> > I have the same problem with you, do u solve it ?
> 
> I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 . On my machine, performance on ssv2(OTAM ) after traing 75000 tasks is 50.5+/-0.4 , perfomance on ucf101(ARN) after training 10000 tasks is 82.0+/-0.3. Have you solved this problem? QAQ

I try the setting with res50 and 224x224, and achive the same accuracy as stated in this paper. I think you should just replace res18 or increase the img size.

**Comment by xiaoc-git** [comment]
I use the code to train on hmdb51 with the suggested hyper-parameters and the same data loader. However, the 5-way 5-shot accuracy can only 74.3%, which is lower than the reported 75.6%. 
Besides,the accuracy on val set is much higher than in the test set,are you the same as me? 
And ,could you give me the correct video_reader document?I will be very grateful.(1842029400@qq.com)

**Comment by M77827** [comment]
> > > I have the same problem with you, do u solve it ?
> > 
> > 
> > I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 . On my machine, performance on ssv2(OTAM ) after traing 75000 tasks is 50.5+/-0.4 , perfomance on ucf101(ARN) after training 10000 tasks is 82.0+/-0.3. Have you solved this problem? QAQ
> 
> I try the setting with res50 and 224x224, and achive the same accuracy as stated in this paper. I think you should just replace res18 or increase the img size.

佬 ，你处理数据集是根据作者另一个仓库中给到的代码来处理的吗

**Comment by M77827** [comment]
> > > I have the same problem with you, do u solve it ?
> > 
> > 
> > I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 . On my machine, performance on ssv2(OTAM ) after traing 75000 tasks is 50.5+/-0.4 , perfomance on ucf101(ARN) after training 10000 tasks is 82.0+/-0.3. Have you solved this problem? QAQ
> 
> I try the setting with res50 and 224x224, and achive the same accuracy as stated in this paper. I think you should just replace res18 or increase the img size.

哥，能不能分享一下你用的数据集，我所有的实验设置都和作者的一样，但是结果还是差点，就感觉应该是数据集处理有问题。

**Comment by MIGI9527** [comment]
> > > > I have the same problem with you, do u solve it ?
> > > 
> > > 
> > > I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 . On my machine, performance on ssv2(OTAM ) after traing 75000 tasks is 50.5+/-0.4 , perfomance on ucf101(ARN) after training 10000 tasks is 82.0+/-0.3. Have you solved this problem? QAQ
> > 
> > 
> > I try the setting with res50 and 224x224, and achive the same accuracy as stated in this paper. I think you should just replace res18 or increase the img size.
> 
> 佬 ，你处理数据集是根据作者另一个仓库中给到的代码来处理的吗

哥们能不能分享一下你的数据集或者告诉一下怎么处理数据集的，能不能加个联系方式

**Comment by M77827** [comment]
> > > > > I have the same problem with you, do u solve it ?
> > > > 
> > > > 
> > > > I've met the same troubles , and my machine can only support the options of resnet18 with img size 84x84 . On my machine, performance on ssv2(OTAM ) after traing 75000 tasks is 50.5+/-0.4 , perfomance on ucf101(ARN) after training 10000 tasks is 82.0+/-0.3. Have you solved this problem? QAQ
> > > 
> > > 
> > > I try the setting with res50 and 224x224, and achive the same accuracy as stated in this paper. I think you should just replace res18 or increase the img size.
> > 
> > 
> > 佬 ，你处理数据集是根据作者另一个仓库中给到的代码来处理的吗
> 
> 哥们能不能分享一下你的数据集或者告诉一下怎么处理数据集的，能不能加个联系方式

可以 先邮箱M7628455@163.com
